src/olgm/herald/Feature.js

```diff
@@ -232,18 +234,16 @@ class FeatureHerald extends Herald {
       const index = zIndex !== undefined ? zIndex : this.index_;
 
       const image = style.getImage();
-      const useCanvas = this.mapIconOptions_.useCanvas !== undefined ?
-        this.mapIconOptions_.useCanvas : false;
-      if (image && image instanceof Icon && useCanvas) {
-        this.marker_ = createMapIcon(image, latLng, index);
+      if (image && image instanceof Icon && mapIconOptions.useCanvas) {
+        this.marker_ = createMapIcon(image, latLng, index, this.feature_.usePane);
```

Comment:
the getter would ge used there:

```
this.feature_.usePane
```

would become

```
this.feature_.usePane
```

by this:

```
this.feature_.get('olgm_pane')
```

What do you think?

Reply:
Yep, updated.